ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1239,8 +1239,8 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         return imageView
     }()
 
-    internal let topSeparator = MSFDivider()
-    internal let bottomSeparator = MSFDivider()
+    internal let topSeparator = Separator(orientation: .horizontal)
```

Comment:
nit: Can these just be `Separator()`?

Reply:
Probably, but I don't want to change that value during a merge.